lambda/lambda.js

```diff
@@ -99,6 +99,11 @@ async function handler(request) {
                     resolve(false);
                     return;
                 }
+                if (Math.floor(Date.now() / 1000) + 3600 < payload.exp) {
```

Comment:
It would be nice to include a comment here explaining that `3600` (e.g. the information you've collected from the Space and JBA teams)